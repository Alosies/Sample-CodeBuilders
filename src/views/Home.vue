<template>
  <div class="container">
    <Events />
    <Posts />
  </div>
</template>

<script>
import Events from '../components/Events'
import Posts from '../components/BlogPosts'

export default {
  components: {
    Events,
    Posts
  }
}
</script>

<style lang="stylus" scoped>
.container {
  display: grid;
  grid-template-columns: repeat(auto-fit, minmax(30rem, 1fr));
  grid-gap: 2rem;
}
</style>
