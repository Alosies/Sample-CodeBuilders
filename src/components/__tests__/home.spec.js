import { shallowMount } from '@vue/test-utils'
import Home from '@/views/Home.vue'
import Vuetify from 'vuetify'
import Vue from 'vue'

describe('HelloWorld.vue', () => {
  Vue.use(Vuetify)
  it('Home Component renders', () => {
    const wrapper = shallowMount(Home)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
