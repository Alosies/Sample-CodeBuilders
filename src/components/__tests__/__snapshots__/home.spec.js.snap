// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue Home Component renders 1`] = `
<div class="container">
  <events-stub></events-stub>
  <posts-stub></posts-stub>
</div>
`;
